source/operations/monitoring.rst

```diff
@@ -16,9 +16,7 @@ Monitoring and Alerts
 Metrics and Alerts
 ------------------
 
-MinIO provides point-in-time metrics on cluster status and operations.
-The :ref:`MinIO Console <minio-console-metrics>` provides a graphical display of these metrics.
-
+MinIO publishes a metrics endpoint for scraping data on cluster status and operations.
```

Comment:
Ohh yeah this could be better in light of v3. Maybe refer to "endpoints" here?  
```suggestion
MinIO publishes metrics endpoints for scraping data on cluster status and operations.
```